docs/deployment-process/issue-tracking/azure-devops.md

```diff
@@ -0,0 +1,90 @@
+Octopus does this by using the build information associated with any packages contained in the release to request work item references from Azure DevOps.
+
+To supply the build information:
+
+1. Install one of our official build server plugins with support for our metadata step. This list currently includes [JetBrains TeamCity](https://plugins.jetbrains.com/plugin/9038-octopus-deploy-integration), [Atlassian Bamboo](https://marketplace.atlassian.com/apps/1217235/octopus-deploy-bamboo-add-on?hosting=server&tab=overview), and [Azure DevOps](https://marketplace.visualstudio.com/items?itemName=octopusdeploy.octopus-deploy-build-release-tasks). We're currently working on adding support for the Jenkins plugin.
+
+2. Update your build process to add and configure the Octopus Metadata step.
+
+3. Ensure your Octopus Metadata step has set the 'Process commit messages' or 'Work Items Source' field.
+
+## Connect Octopus to Azure DevOps {#connect-octopus-to-azure-devops}
+
+1. Configure the Azure DevOps Issue Tracker extension in Octopus Deploy.
+
+    In the Octopus web portal, navigate to **{{Configuration,Settings,Azure DevOps Issue Tracker}}** and set the following values:
+
+    - **Azure DevOps Base URL**. This tells Octopus where the Azure DevOps instance is located.
+
+    - Unless the Azure DevOps instance is public, you'll need to supply a **Personal Access Token**, created in the Azure DevOps User Settings (under Personal access tokens), with authorization to read scopes **Build** and **Work items**.
+
+    - Ensure the **Is Enabled** property is set as well.
+
+2. Configure the Release Note Options _(optional)_.
+
+    - **Release Note Prefix**: If specified, Octopus will look for a work item comment that starts with the given prefix text and use whatever text appears after the prefix as the release note, which will be available in the [build information](/docs/packaging-applications/build-servers/index.md#build-information) as the work item's description. If no comment is found with the prefix then Octopus will default back to using the title for that work item.
+
+    For example, a prefix of `Release note:` can be used to identify a customer friendly work item title vs a technical feature or bug fix title.
+
+When configured, this integration will retrieve Azure DevOps work item details and add details to your releases and deployments.
+
+## Troubleshooting
+
+### Error message: "Unable to obtain some work item data."
+
+The rest of the message will have more information about what went wrong, but this typically indicates a failure to do one of the following:
+
+* contact Azure DevOps over the network
+* authenticate using the specified Personal Access Token to read scopes **Build** and **Work items**
+* retrieve build and work item data, and work item comments
+* interpret the data
+
```

Comment:
Would it make sense to offer a suggestion here. Something like `It should be straightforward to resolve most of these issues. For example, if there is a problem retrieving build and work item details, ensure that your Personal Access Token has appropriate access to the right scopes as mentioned in the section [above](link).`

Reply:
Maybe, but how can I avoid sentences like "If it's your network that is broken, try fixing your network" ? It's one thing to speculate about what's wrong, like I am doing in this passage, but I feel like there's too little to go on to turn it into a recommendation.

Reply:
After more discussion with @wordlee I'm removing the section and will add suggestion to error messages where appropriate.